services/commissioning.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rovercode Commissioning Service
+After=syslog.target docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+User=root
+Group=root
+ExecStartPre=-/usr/bin/docker stop rovercode-commissioning
+ExecStart=-/usr/bin/docker run --name rovercode-commissioning --privileged -v /dev:/dev -w='/var/rovercode/commissioning' --entrypoint=/bin/bash rovercode -c 'python app.py /dev/something /opt/rovercode/ TODO: use real /dev location'
```

Comment:
Will the `TODO` be removed later?

Reply:
Yup. I've got to experiment some on the RaspPi to figure out that argument.

Reply:
Removed.